### PR TITLE
Refactor code style, basic corrections

### PR DESCRIPTION
**Pull request by elhanarinc**
* Module.exports vs export [Check here](https://hackernoon.com/import-export-default-require-commandjs-javascript-nodejs-es6-vs-cheatsheet-different-tutorial-example-5a321738b50f)
* Another good reference for exports [Check here](https://stackoverflow.com/questions/34645731/export-more-than-one-variable-in-es6/34645759)

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ app.use(sassMiddleware({
   indentedSyntax: true, // true = .sass and false = .scss
   sourceMap: true,
 }));
+app.use(helmet());
```
Comment (ardaasln) [inline src/app.js]: Use `import helmet from 'helmet';` at the top

File: src/routes/index.js [hunk 2]
```diff
@@ -6,4 +6,8 @@ router.get('/', function(req, res, next) {
   res.render('index', {title: 'Express'});
 });
 
-module.exports = router;
+/**
+ * Exports.
+ */
+
+export {router};
```
Comment (ardaasln) [inline src/routes/index.js]: Also fix app.js
-let index = require('./routes/index');
+import {router as index} from './routes/index';

File: src/routes/users.js [hunk 3]
```diff
@@ -6,4 +6,8 @@ router.get('/', function(req, res, next) {
   res.send('respond with a resource');
 });
 
-module.exports = router;
+/**
+ * Exports.
+ */
+
+export {router};
```
Comment (ardaasln) [inline src/routes/users.js]: Also fix app.js
-let users = require('./routes/users');
+import {router as users} from './routes/users';

File: src/background/init.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      elements.push(i);
+    }
+    async.forEachOfLimit(elements, 2, (elementVal, elementKey, callback) => {
+      url = mainUrl + '?p=' + elementVal;
+      crawler.crawl(url, callback);
+    });
+  });
+}
+
+export {backgroundTask};
```
Comment (ardaasln) [inline src/background/init.js]: Also add clientSecrets to exports
`+export {backgroundTask, clientSecrets};`

File: src/bin/www [hunk 5]
```diff
@@ -2,13 +2,17 @@
...
-import * as fs from 'fs';
-import * as youtube from '../work/youtube.js';
-import app from '../app.js';
-import * as crawler from '../work/crawl.js';
 import * as http from 'http';
-import * as mongo from '../db/mongo.js';
+
+import app from '../app.js';
+
+import backgroundTask as bg from '../background/init.js'
```
Comment (ardaasln) [inline src/bin/www]: Syntax error
`-import backgroundTask as bg from '../background/init.js'`
`+import {backgroundTask as bg} from '../background/init.js'`

File: src/lib/crawl.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ import request from 'request';
 import cheerio from 'cheerio';
 import * as async from 'async';
 import * as youtube from './youtube.js';
-import * as www from '../bin/www.js';
+import clientSecrets from '../background/init.js';
```
Comment (ardaasln) [inline src/lib/crawl.js]: We have now multiple exports at init.js so:
`-import clientSecrets from '../background/init.js';`
`+import {clientSecrets} from '../background/init.js';`

**Review by ardaasln — CHANGES_REQUESTED** [review]
Commented on related code chunks for changes

**Comment by elhanarinc** [comment]
Review done. @ardaasln

**Review by ardaasln — APPROVED** [review]

**Pull request merged**